applications/vessel-history/public/locales/source/translations.json

```diff
@@ -24,19 +24,23 @@
   },
   "vessel": {
     "authorizationPlural": "authorizations",
+    "callsign": "callsign",
+    "flag": "flag",
     "historyLabelByField": "{{label}} history for {{vesselName}}",
+    "imo": "imo",
+    "mmsi": "mmsi",
     "noAuthorizations": "No authorizations found",
     "plusPreviousValuesByField": "+{{quantity}} previous {{fieldLabel}}",
     "saveForOfflineView": "save vessel for offline view",
+    "source": "source",
+    "transmissionPlural": "transmissions",
```

Comment:
I think this is wrong.. https://www.i18next.com/translation-function/plurals

Reply:
Fixed